### PR TITLE
PLATFORM-1762 Create get_services_identifiers_by_tag_name method at container

### PR DESCRIPTION
**Pull request by jorgemo-fever**
PLATFORM-1762

#### Description
Tags would get advantage of this functionality and it is needed for implementing it at Transactional API.

#### How to test
- [ ] Locally install this version of pypendency.
- [ ] Register a service and then call the new method to obtain the services identifiers
- [ ] Test also the edge case: service not registered

### CODE REVIEW

File: src/pypendency/container.py [hunk 1]
```diff
@@ -73,39 +73,15 @@ def set(self, identifier: str, service: object, tags: Optional[Set[Tag]] = None)
             for tag in tags:
                 self.__add_service_to_tag_group(tag, identifier)
 
+    def has(self, identifier: str) -> bool:
```
Comment (gprossignoli) [inline src/pypendency/container.py:76]: has_identifier_registered or smth more meaningful?

Comment (jorgemo-fever) [reply src/pypendency/container.py:76]: This function existed previously, changing it would break the API. I only moved it down because of call order

File: src/pypendency/container.py [hunk 2]
```diff
@@ -149,5 +125,43 @@ def __instance_from_fqn(self, fully_qualified_name: str, args: list, kwargs: dic
         except TypeError as e:
             raise exceptions.ServiceInstantiationFailed(fully_qualified_name) from e
 
-    def has(self, identifier: str) -> bool:
-        return identifier in self._service_mapping
+    def get_service_identifier(self, service: Union[object, Definition]) -> str:
```
Comment (gprossignoli) [inline src/pypendency/container.py:128]: So this isn't new, it was moved, right?

**Review by gprossignoli — APPROVED** [review]
